code/espurna/config/general.h

```diff
@@ -839,14 +1076,32 @@ PROGMEM const char* const custom_reset_string[] = {
 #define NTP_SUPPORT             1               // Build with NTP support by default (6.78Kb)
 #endif
 
+#ifndef NTP_SERVER
 #define NTP_SERVER              "pool.ntp.org"  // Default NTP server
+#endif
+
+#ifndef NTP_TIMEOUT
```

Comment:
Besides timeout guards both NTP_TIME_OFFSET and NTP_DAY_LIGHT

Reply:
done.